event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/feed/eventstream/EventStreamEntry.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.services.eventsourcing.source.api.feed.eventstream;
+
+import java.util.UUID;
+
+public class EventStreamEntry {
+    private final String streamId;
+    private final String href;
+
+    public EventStreamEntry(final UUID streamId, final String href) {
```

Comment:
This is a dto object to be serialised into json. Nothing to do with db.
I just want to make sure that the UUID gets serialised to json properly.